CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ See `PUBLISH.md` for instructions on how to publish a new version.
 
 - (minor) Add syntax for defining settings on images, such as size and alignment
 - (patch) Dependency updates
+- (minor) Add hash links to heading_id plugin
```

Comment:
Sorry, I realise now this is my terrible note writing in this file, but can you add this to the start of the list (above images), not the end?